x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -62,17 +77,17 @@ function useServicesFetcher() {
         }).then((mainStatisticsData) => {
           return {
             requestId: uuid(),
```

Comment:
I know you didn't add this but wouldn't it be better if `useFetcher` added a unique id to each response.

```ts
const mainStatisticsFetch = useFetcher();

// ...

const { id, data, status, error }  = mainStatisticsFetch;
```